internal/signerverifier/dsse/dsse_test.go

```diff
@@ -52,9 +82,9 @@ func createSignedEnvelope() (*d.Envelope, error) {
 		Signatures:  []d.Signature{},
 	}
 
-	env, err = SignEnvelope(context.Background(), env, privateKeyBytes)
+	env, err = SignEnvelope(context.Background(), env, signer)
```

Comment:
plumb context

Reply:
This is a test. SignEnvelope and VerifyEnvelope accept context already.